Vagrantfile

```diff
@@ -194,14 +206,17 @@ Vagrant.configure("2") do |config|
     },
     ubuntu: {
       name: 'ubuntu/bionic64',
-      version: '20190514.0.0'
+      version: '20190514.0.0',
+      scripts: [
+        {
+          name: "install-python",
+          type: "shell",
+          data: INSTALL_PYTHON
```

Comment:
Since the script is a one-liner, you could write `data: 'apt install python -y'` directly.

Reply:
We use the same for regular Ubuntu nodes too